resources/public/api/conf/1.0/application.yaml

```diff
@@ -2181,3 +2181,14 @@ components:
           refreshUrl: 'https://api.service.hmrc.gov.uk/oauth/refresh'
           scopes:
             common-transit-convention-traders: Access all aspects of CTC Traders API
+    applicationRestricted:
+      type: oauth2
+      description: >-
+        HMRC supports OAuth 2.0 for authenticating application restricted API 
+        requests using an OAuth 2.0 Bearer Token in the AUTHORIZATION header.
+        See https://developer.service.hmrc.gov.uk/api-documentation/docs/authorisation/application-restricted-endpoints for details.
+      flows:
+        clientCredentials:
+          tokenUrl: https://api.service.hmrc.gov.uk/oauth/token
+          scopes:
+            common-transit-convention-traders: Access push-pull-notification box information via the CTC Traders API

```

Comment:
"Via" is a Latinate word. Can we replace with an alternative, such as "through" or "by using"?